.travis.yml

```diff
@@ -1,6 +1,10 @@
 language: java
 
 sudo: false
+dist: precise
+
+before_install:
+  - export M2_HOME=/usr/local/maven
```

Comment:
We may install maven manually instead of using the default one. But disadvantage is that we can only use maven with a specific version. Do you think it necessary?